openshift_tools/python-openshift-tools.spec

```diff
@@ -38,27 +38,96 @@ cp -p web/*.py %{buildroot}%{python_sitelib}/openshift_tools/web
 # openshift_tools files
 %files
 %dir %{python_sitelib}/openshift_tools
+%dir %{python_sitelib}/openshift_tools/monitoring
+%{python_sitelib}/openshift_tools/monitoring/__init*
 %{python_sitelib}/openshift_tools/*.py
 %{python_sitelib}/openshift_tools/*.py[co]
 
 
+# ----------------------------------------------------------------------------------
+# python-openshift-tools-monitoring-pcp subpackage
+# ----------------------------------------------------------------------------------
+%package monitoring-pcp
+Summary:       OpenShift Tools PCP Python Library Package
+Requires:      python2,python-openshift-tools,python-pcp
+BuildArch:     noarch
 
+%description monitoring-pcp
+PCP Python libraries developed for monitoring OpenShift.
+
+%files monitoring-pcp
+%{python_sitelib}/openshift_tools/monitoring/pminfo*.py
+%{python_sitelib}/openshift_tools/monitoring/pminfo*.py[co]
 
 # ----------------------------------------------------------------------------------
-# python-openshift-tools-monitoring subpackage
+# python-openshift-tools-monitoring-docker subpackage
 # ----------------------------------------------------------------------------------
-%package monitoring
-Summary:       OpenShift Tools Monitoring Python Package
-Requires:      python2,python-openshift-tools,python-zbxsend,python-openshift-tools-web,python-pcp,atomic-openshift,python-awscli
+%package monitoring-docker
+Summary:       OpenShift Tools Docker Python Libraries Package
+Requires:      python2,python-openshift-tools,python-pcp
 BuildArch:     noarch
 
-%description monitoring
-Tools developed for monitoring OpenShift.
+%description monitoring-docker
+Docker Python libraries developed for monitoring OpenShift.
 
-%files monitoring
-%dir %{python_sitelib}/openshift_tools/monitoring
-%{python_sitelib}/openshift_tools/monitoring/*.py
-%{python_sitelib}/openshift_tools/monitoring/*.py[co]
+%files monitoring-docker
+%{python_sitelib}/openshift_tools/monitoring/dockerutil.py
+%{python_sitelib}/openshift_tools/monitoring/dockerutil.py[co]
+
+
+
+# ----------------------------------------------------------------------------------
+# python-openshift-tools-monitoring-zagg subpackage
+# ----------------------------------------------------------------------------------
+%package monitoring-zagg
+Summary:       OpenShift Tools Zagg Python Libraries Package
+Requires:      python2,python-openshift-tools,python-openshift-tools-web,python-zbxsend
+BuildArch:     noarch
+
+%description monitoring-zagg
+Zagg Python libraries developed for monitoring OpenShift.
+
+%files monitoring-zagg
+%{python_sitelib}/openshift_tools/monitoring/metricmanager.py
+%{python_sitelib}/openshift_tools/monitoring/metricmanager.py[co]
+%{python_sitelib}/openshift_tools/monitoring/zagg*.py
+%{python_sitelib}/openshift_tools/monitoring/zagg*.py[co]
+%{python_sitelib}/openshift_tools/monitoring/zbxapi.py
+%{python_sitelib}/openshift_tools/monitoring/zbxapi.py[co]
```

Comment:
I think that zbxapi.py* needs to be in it's own package. Possibly named python-openshift-tools-monitoring-zabbix.

That way, we can just install zbxapi on boxes where we want to use the ansible module.

This really has nothing to do with zagg, other than the fact that zagg uses it.